dev_tools/conf/mypy.ini

```diff
@@ -7,22 +7,21 @@ follow_imports = silent
 ignore_missing_imports = true
 
 # 3rd-party libs for which we don't have stubs
-[mypy-apiclient.*,freezegun.*,matplotlib.*,mpl_toolkits,multiprocessing.dummy,oauth2client.*,pandas.*,proto.*,pytest.*,scipy.*,sortedcontainers.*,setuptools.*,pylatex.*,networkx.*,qiskit.*,pypandoc.*,ply.*,_pytest.*,google.api.*,google.api_core.*,grpc.*,google.auth.*,google.oauth2.*,google.protobuf.text_format.*,quimb.*,pyquil.*,google.cloud.*,filelock.*,codeowners.*,tqdm.*,importlib_metadata.*,google.colab.*,IPython.*,astroid.*,pylint.*]
+
+# Google
+[mypy-google.api_core.*,,google.auth.*,google.colab.*,google.protobuf.text_format.*,google.protobuf.*,google.cloud.*]
+follow_imports = silent
+ignore_missing_imports = true
+
+# Non-Google
+[mypy-sympy.*,matplotlib.*,proto.*,pandas.*,scipy.*,freezegun.*,mpl_toolkits.*,networkx.*,ply.*,astroid.*,pytest.*,_pytest.*,pylint.*,setuptools.*,qiskit.*,quimb.*,pylatex.*,filelock.*,sortedcontainers.*,tqdm.*]
 follow_imports = silent
 ignore_missing_imports = true
 
-# There was no type information before numpy 1.20, so there are numpy mypy issues in the codebase
-[mypy-numpy.*]
-follow_imports = skip
-follow_imports_for_stubs = true
 
 # Treat symbols imported from Google's protobuf library as type Any.
-# This supresses errors due to attributes not known to typeshed,
+# This suppresses errors due to attributes not known to typeshed,
 # e.g. Descriptor._options.
 [mypy-google.protobuf.*]
 follow_imports = skip
 follow_imports_for_stubs = true
```

Comment:
Perhaps we can move this under the first Google block?

Reply:
it has a slightly different structure for the flags it is changing